Formula/terraform-docs.rb

```diff
@@ -1,82 +1,24 @@
-require "language/go"
-
 class TerraformDocs < Formula
   desc "Tool to generate documentation from Terraform modules"
   homepage "https://github.com/segmentio/terraform-docs"
-  url "https://github.com/segmentio/terraform-docs/archive/v0.3.0.tar.gz"
-  sha256 "0cfac8ed50a6ba458ec5177e493fd8adc05395f3d9ba79504dc33ce6e5733fcd"
-  head "https://github.com/segmentio/terraform-docs.git"
-
-  bottle do
```

Comment:
The bottle block should not be edited manually, please restore it.

Reply:
Thanks for the feedback. After going through the documentation, I had assumed that your Jenkins pipeline would automatically insert this block. The block has been restored with https://github.com/Homebrew/homebrew-core/pull/32799/commits/06479f68003572d26b6327aceb78dcd048972209.